applications/StructuralMechanicsApplication/structural_mechanics_application_variables.h

```diff
@@ -154,6 +154,8 @@ namespace Kratos
     KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS(STRUCTURAL_MECHANICS_APPLICATION, MIDDLE_VELOCITY)
     KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS(STRUCTURAL_MECHANICS_APPLICATION, MIDDLE_ANGULAR_VELOCITY)
     KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS(STRUCTURAL_MECHANICS_APPLICATION, NODAL_INERTIA)
+    KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, NODAL_DISPLACEMENT_DAMPING)
+    KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS(STRUCTURAL_MECHANICS_APPLICATION, NODAL_ROTATIONAL_DAMPING)
```

Comment:
can you call it `NODAL_ROTATION_DAMPING` in order to be consistent? All other variables that have "rotation" in the name are using "ROTATION"